caim_base/notifications.py

```diff
@@ -1,5 +1,8 @@
 from templated_email import send_templated_mail
```

Comment:
not a blocker, but since the emails are templated based off the data types we have currently in django, would be good to have just one or two sanity tests that make sure that they fail if we change the data format the email accepts

Reply:
gonna add these tests after merging, so I don't run into merge conflicts/replicate work @Johnetordoff did